src/Control/Carrier/Catch.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE FlexibleInstances, GeneralizedNewtypeDeriving, MultiParamTypeClasses, RankNTypes, TypeOperators, UndecidableInstances #-}
+
+-- | A carrier for a 'Catch' effect.
+module Control.Carrier.Catch
```

Comment:
(If we rename the `Catch` effect to `CatchIO` we should mirror that here as well.)

Reply:
Or maybe Control.Carrier.Exception.Catch ?

Reply:
I could live with that if we rename the effect to `Exception`, totally.